samples/dotnetbot/Program.cs

```diff
@@ -6,23 +6,23 @@ public static class Program
     {
                 public static void Main(string[] args) 
                 {
-                        string message = "";
-                        if (args.Length < 1) 
-                        {
-                          message = "Welcome to .NET Core!";
-                        } else 
+                      string message = "Welcome to .NET Core!";
+                        
+                       if (args.Length > 0) 
                         {
+                          message = String.Empty;
                           foreach (string item in args) 
```

Comment:
This here was done because of CoreRT. Since this is now a pure IL app, you could do something like `String.Join(args, ' ')` to get the final message to display. :)